src/mongo/db/pipeline/TDigest.cpp

```diff
@@ -118,12 +120,12 @@ TDigest TDigest::merge(Range<const double*> unsortedValues) const {
   std::copy(unsortedValues.begin(), unsortedValues.end(), in);
 
   detail::double_radix_sort(n, buckets.get(), in, out);
-  DCHECK(std::is_sorted(in, in + n));
+  //DCHECK(std::is_sorted(in, in + n));
 
-  return merge(presorted, Range<const double*>(in, in + n));
+  return merge(presorted, std::vector<double>(in, in + n));
 }
 
-TDigest TDigest::merge(presorted_t, Range<const double*> sortedValues) const {
+TDigest TDigest::merge(presorted_t, const std::vector<double> & sortedValues) const {
```

Comment:
If the other method is removed we can simplify the signature here a little to
`TDigest TDigest::merge(const std::vector<double> &sortedValues) const`

So we get rid of this `presorted_t` type.